source/Distribution.h

```diff
@@ -0,0 +1,41 @@
+/* Distribution.h
+Copyright (c) 2022 by DeBlister
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#ifndef DISTRIBUTION_H_
+#define DISTRIBUTION_H_
+
+#include "Angle.h"
+
+#include <tuple>
+
+
+
+class Distribution{
+public:
+	enum class Type{
+		Tight = 0,
+		Middling = 1,
+		Wide = 2,
+		Uniform = 3,
+		Triangular = 4
```

Comment:
You don't need assign values to the enum members if they start from 0 and are always one more than the one before.

Reply:
I rather prefer the explicit method since they correspond to a lookup table.